src/backend/bigquery/bigquery.ts

```diff
@@ -257,7 +257,7 @@ const buildSqlQuery = function (
 
```

Comment:
I find the refactored version of this code less readable.  Is it mainly to use `null` instead of `''`, or are there other reasons for it?